{"kind":"symbol","primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Linux"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":" { ","kind":"text"},{"text":"get","kind":"keyword"},{"text":" }","kind":"text"}],"languages":["swift"]}]}],"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection"]]},"abstract":[{"type":"text","text":"The location where the error was thrown."}],"sections":[],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/location"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/location"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"modules":[{"name":"NIOHTTP2"}],"externalID":"s:8NIOHTTP20A6ErrorsO20IOOnClosedConnectionV8locationSSvp","roleHeading":"Instance Property","title":"location","symbolKind":"property","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"location"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"}]},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection/location":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection\/location","role":"symbol","url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection\/location","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"location","kind":"identifier"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"}],"title":"location","abstract":[{"type":"text","text":"The location where the error was thrown."}],"kind":"symbol","type":"topic"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","role":"collection","title":"NIOHTTP2","url":"\/documentation\/niohttp2","abstract":[],"kind":"symbol","type":"topic"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors/IOOnClosedConnection":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors\/IOOnClosedConnection","title":"NIOHTTP2Errors.IOOnClosedConnection","abstract":[{"type":"text","text":"An attempt to perform I\/O was made on a connection that is already closed."}],"kind":"symbol","type":"topic","role":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"IOOnClosedConnection"}],"navigatorTitle":[{"kind":"identifier","text":"IOOnClosedConnection"}],"url":"\/documentation\/niohttp2\/niohttp2errors\/ioonclosedconnection"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/NIOHTTP2Errors":{"navigatorTitle":[{"text":"NIOHTTP2Errors","kind":"identifier"}],"type":"topic","title":"NIOHTTP2Errors","abstract":[{"type":"text","text":"Errors that NIO raises when handling HTTP\/2 connections."}],"url":"\/documentation\/niohttp2\/niohttp2errors","kind":"symbol","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NIOHTTP2Errors","kind":"identifier"}],"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/NIOHTTP2Errors"}}}